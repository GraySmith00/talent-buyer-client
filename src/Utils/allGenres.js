export const allGenres = [
  'big room',
  'dance pop',
  'edm',
  'electro house',
  'pop',
  'progressive electro house',
  'tropical house',
  'house',
  'minimal techno',
  'schranz',
  'tech house',
  'techno',
  'electropop',
  'indie r&b',
  'indietronica',
  'deep big room',
  'dutch house',
  'progressive house',
  'folk',
  'folk-pop',
  'lilith',
  'mellow gold',
  'new wave pop',
  'pop rock',
  'scottish new wave',
  'soft rock',
  'alternative dance',
  'australian dance',
  'filter house',
  'big beat',
  'disco house',
  'electronic',
  'new rave',
  'uk garage',
  'vocal house',
  'aussietronica',
  'indie poptimism',
  'vapor soul',
  'progressive uplifting trance',
  'uplifting trance',
  'art pop',
  'chamber pop',
  'experimental pop',
  'icelandic pop',
  'metropopolis',
  'permanent wave',
  'trip hop',
  'afro house',
  'deep house',
  'kwaito house',
  'bass trap',
  'brostep',
  'dubstep',
  'electronic trap',
  'german techno',
  'microhouse',
  'contemporary country',
  'country',
  'country road',
  'modern country rock',
  'redneck',
  'brazilian electronica',
  'deep groove house',
  'melbourne bounce',
  'tropical pop edm',
  'deep disco house',
  'deep melodic euro house',
  'minimal tech house',
  'tribal house',
  'chamber psych',
  'float house',
  'shiver pop',
  'downtempo',
  'hip hop',
  'turntablism',
  'adult standards',
  'cabaret',
  'christmas',
  'easy listening',
  'lounge',
  'swing',
  'vocal jazz',
  'catstep',
  'complextro',
  'filthstep',
  'progressive trance',
  'trance',
  'bass music',
  'future garage',
  'dance-punk',
  'electroclash',
  'neo-synthpop',
  'nu jazz',
  'nu skool breaks',
  'australian hip hop',
  'deep australian indie',
  'escape room',
  'chillwave',
  'indie psych-rock',
  'nu disco',
  'shimmer pop',
  'vaporwave',
  'wonky',
  'acid house',
  'chicago house',
  'hip house',
  'laboratorio',
  'new romantic',
  'new wave',
  'vintage french electronic',
  'deep minimal techno',
  'breakbeat',
  'hardcore techno',
  'norwegian hip hop',
  'norwegian pop',
  'vapor twitch',
  'deep tropical house',
  'ccm',
  'christian alternative rock',
  'christian music',
  'christian rock',
  'world worship',
  'worship',
  'indie pop',
  'indie rock',
  'modern rock',
  'swedish electropop',
  'swedish indie pop',
  'swedish synthpop',
  'etherpop',
  'gauze pop',
  'drum and bass',
  'liquid funk',
  'intelligent dance music',
  'ambient',
  'art rock',
  'compositional ambient',
  'dream pop',
  'drone',
  'experimental',
  'experimental rock',
  'fluxwork',
  'fourth world',
  'freak folk',
  'hauntology',
  'mandible',
  'neo-psychedelic',
  'noise pop',
  'noise rock',
  'outsider house',
  'post-rock',
  'glitch hop',
  'banda',
  'grupera',
  'norteno',
  'ranchera',
  'regional mexican',
  'dance rock',
  'europop',
  'synthpop',
  'post-teen pop',
  'strut',
  'acid techno',
  'detroit techno',
  'dub techno',
  'modern alternative rock',
  'chillstep',
  'tracestep',
  'hardstyle',
  'electro',
  'belgian indie',
  'mashup',
  'neurofunk',
  'acid jazz',
  'danish electro-pop',
  'finnish edm',
  'deep euro house',
  'vapor pop',
  'indie anthem-folk',
  'indie electro-pop',
  'viral pop',
  'indian indie',
  'pop house',
  'pop rap',
  'alternative rock',
  'garage rock',
  'rock',
  'electro swing',
  'indie dream pop',
  'country gospel',
  'country rock',
  'cowboy western',
  'nashville sound',
  'outlaw country',
  'traditional country',
  'northern irish indie',
  'latvian pop',
  'dutch hip hop',
  'dark jazz',
  'grave wave',
  'destroy techno',
  'rap',
  'underground hip hop',
  'chillhop',
  'jazz boom bap',
  'la indie',
  'stomp and holler',
  'r&b',
  'british jazz',
  'broken beat',
  'indie jazz',
  'moombahton',
  'zapstep',
  'wave',
  'candy pop',
  'scottish hip hop',
  'australian alternative rock',
  'japanese classical performance',
  'violin',
  'australian pop',
  'eurodance',
  'neo soul',
  'british contemporary classical',
  'british soundtrack',
  'soundtrack',
  'swedish pop',
  'emo rap',
  'krautrock',
  'austindie',
  'gangster rap',
  'southern hip hop',
  'trap music',
  'abstract beats',
  'arab alternative',
  'brooklyn indie',
  'dreamo',
  'french indietronica',
  'jam band',
  'pop punk',
  'alternative hip hop',
  'east coast hip hop',
  'hardcore hip hop',
  'melancholia',
  'uk contemporary r&b',
  'britpop',
  'madchester',
  'post-punk',
  'uk post-punk',
  'athens indie',
  'indie emo'
];
