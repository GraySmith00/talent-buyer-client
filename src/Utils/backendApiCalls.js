import axios from 'axios';
import { artistsByAgency } from './artistsByAgency';
import { getArtistInfo } from './songKickApiCalls';

export const signUpPostRequest = async user => {
  const url = `${process.env.REACT_APP_API}/signup`;
  const response = await axios.post(url, user);
  return response.data;
};

export const logInPostRequest = async userCreds => {
  const url = `${process.env.REACT_APP_API}/login`;
  const response = await axios.post(url, userCreds);
  if (response.status === 200) {
    return response;
  }
};

export const venuePostRequest = async venue => {
  const url = `${process.env.REACT_APP_API}/api/v1/venues`;
  const response = await axios.post(url, venue);
  const data = response.data;
  return data;
};

export const getAllUserVunues = async userId => {
  const url = `${process.env.REACT_APP_API}/api/v1/buyers/${userId}/venues`;
  const response = await axios.get(url);
  const data = response.data;
  return data;
};

export const artistPostRequest = async artist => {
  const url = `${process.env.REACT_APP_API}/api/v1/artists`;
  await fetch(url, {
    method: 'POST',
    body: JSON.stringify(artist),
    headers: {
      'Content-Type': 'application/json'
    }
  });
};

export const addWmeArtistsToBackend = () => {
  artistsByAgency.WME.map(async artistName => {
    const artistInfo = await getArtistInfo(artistName, 'WME');
    if (artistInfo) {
      await artistPostRequest(artistInfo);
    }
  });
};

export const addCaaArtistsToBackend = () => {
  artistsByAgency.CAA.map(async artistName => {
    const artistInfo = await getArtistInfo(artistName, 'CAA');
    if (artistInfo) {
      await artistPostRequest(artistInfo);
    }
  });
};

export const getAllArtists = async () => {
  const url = `${process.env.REACT_APP_API}/api/v1/artists`;
  const artists = await axios.get(url);
  return artists.data;
};

export const getArtist = async id => {
  const url = `${process.env.REACT_APP_API}/api/v1/artists/${id}`;
  const artist = await axios.get(url);
  return artist.data;
};

export const offerPostRequest = async offer => {
  const url = `${process.env.REACT_APP_API}/api/v1/offers`;
  const savedOffer = await axios.post(url, offer);
  return savedOffer.data;
};

export const getAllUserOffers = async userId => {
  const url = `${process.env.REACT_APP_API}/api/v1/buyers/${userId}/offers`;
  const response = await axios.get(url);
  return response.data;
};

export const offerPutRequest = async offer => {
  const url = `${process.env.REACT_APP_API}/api/v1/buyers/${
    offer.buyer_id
  }/offers/${offer.id}`;
  const response = await axios.put(url, offer);
  return response.data;
};

export const watchlistPostRequest = async artistId => {
  const url = `${process.env.REACT_APP_API}/api/v1/favorite_artists`;
  const response = await axios.post(url, { artist_id: artistId });
  return response;
};

export const watchlistDeleteRequest = async artistId => {
  const url = `${
    process.env.REACT_APP_API
  }/api/v1/favorite_artists/${artistId}`;
  const response = await axios.delete(url);
  return response;
};

export const getUserWatchlist = async () => {
  const url = `${process.env.REACT_APP_API}/api/v1/favorite_artists`;
  const response = await axios.get(url);
  return response;
};
