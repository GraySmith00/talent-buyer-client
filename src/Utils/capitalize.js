export const capitalize = string => {
  return string
    .split(' ')
    .map(word => word.slice(0, 1).toUpperCase() + word.slice(1))
    .join(' ');
};

export const capitalizeArtists = searchResults => {
  if (searchResults.length) {
    const results = searchResults.map(artist => {
      let cleanedArtists = [];
      const split = artist.split(' ');
      split.forEach(name => {
        cleanedArtists.push(name[0].toUpperCase() + name.slice(1));
      });
      return cleanedArtists.join(' ');
    });
    return results;
  }
};
