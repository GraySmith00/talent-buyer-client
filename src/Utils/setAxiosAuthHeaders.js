import axios from 'axios';

const setAxiosAuthHeaders = token => {
  if (token) {
    axios.defaults.headers.common['Authorization'] = token;
  } else {
    delete axios.defaults.headers.common['Authorization'];
  }
};

export default setAxiosAuthHeaders;
