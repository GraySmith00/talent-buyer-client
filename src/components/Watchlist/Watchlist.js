import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import './Watchlist.css';

class Watchlist extends Component {
  render() {
    const { watchlist } = this.props;
    let displayArtists;

    if (!watchlist) {
      displayArtists = <p>Loading...</p>;
    } else {
      displayArtists = watchlist.map(artist => (
        <div className="table-row" key={artist.id}>
          <img src={artist.image_url} alt="artist" className="artist-image" />
          <h3>{artist.name}</h3>
          <h3>{artist.agency}</h3>
          <h3>{artist.popularity}</h3>
        </div>
      ));
    }

    return (
      <div className="watchlist">
        <h2>Watchlist</h2>
        <div className="recent-table-headings">
          <h3>Image</h3>
          <h3>Artist</h3>
          <h3>Agency</h3>
          <h3>Popularity</h3>
        </div>
        <div className="offers-container">{displayArtists}</div>
      </div>
    );
  }
}

Watchlist.propTypes = {
  offers: PropTypes.array.isRequired,
  openEditOfferModal: PropTypes.func.isRequired
};

export const mapStateToProps = state => ({
  watchlist: state.watchlist
});

export default connect(mapStateToProps)(Watchlist);
