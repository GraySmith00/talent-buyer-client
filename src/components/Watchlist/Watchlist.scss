.watchlist {
  .recent-table-headings {
    display: grid;
    grid-template-columns: repeat(5, 10fr);
    border-bottom: 1px solid #fff;
    align-items: center;
    padding-left: 2rem;
    margin: 0;
    h3 {
      margin-bottom: 5px;
    }
  }

  .offers-container {
    height: 140px;
    overflow: scroll;
    background-color: #32333e;
  }

  .artist-image {
    width: 40px;
    height: 40px;
  }

  .table-row {
    display: grid;
    grid-template-columns: repeat(5, 10fr);
    align-items: center;
    justify-content: center;
    margin-left: 2rem;
    font-weight: 300;
    font-size: 0.7rem;
  }
}
