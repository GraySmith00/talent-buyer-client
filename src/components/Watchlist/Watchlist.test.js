import React from 'react';
import { shallow } from 'enzyme';

import { Watchlist } from './Watchlist';
import { mockWatchlist } from '../../Utils/mockData';

describe('Watchlist component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Watchlist watchlist={mockWatchlist} />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
