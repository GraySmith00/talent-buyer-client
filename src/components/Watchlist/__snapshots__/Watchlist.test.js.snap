// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Watchlist component should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Watchlist
    watchlist={
        Array [
            Object {
              "agency": "WME",
              "id": 4,
              "image_url": "https://i.scdn.co/image/021f014a6809e5b7f1ef1c18bf247fd9ccacb193",
              "name": "Alesso",
              "popularity": 81,
              "songkick_id": 4329851,
              "spotify_followers": 2296228,
              "spotify_id": "4AVFqumd2ogHFlRbKIjp1t",
              "spotify_url": "https://open.spotify.com/artist/4AVFqumd2ogHFlRbKIjp1t",
            },
            Object {
              "agency": "WME",
              "id": 18,
              "image_url": "https://i.scdn.co/image/8c0fbe0fb1c86825adeb18bf0a019a22827c4422",
              "name": "Booka Shade",
              "popularity": 51,
              "songkick_id": 295932,
              "spotify_followers": 113010,
              "spotify_id": "2CKaDZ1Yo8YnWega9IeUzB",
              "spotify_url": "https://open.spotify.com/artist/2CKaDZ1Yo8YnWega9IeUzB",
            },
            Object {
              "agency": "WME",
              "id": 23,
              "image_url": "https://i.scdn.co/image/1cbc13e4131461d8631d527c955ba83dddde099d",
              "name": "Chuckie",
              "popularity": 51,
              "songkick_id": 937055,
              "spotify_followers": 100998,
              "spotify_id": "4x7gxsrTH3gThvSKZPPwaQ",
              "spotify_url": "https://open.spotify.com/artist/4x7gxsrTH3gThvSKZPPwaQ",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          Watchlist
</h2>,
        <div
          className="recent-table-headings"
>
          <h3>
                    Image
          </h3>
          <h3>
                    Artist
          </h3>
          <h3>
                    Agency
          </h3>
          <h3>
                    Popularity
          </h3>
</div>,
        <div
          className="offers-container"
>
          <div
                    className="table-row"
          >
                    <img
                              alt="artist"
                              className="artist-image"
                              src="https://i.scdn.co/image/021f014a6809e5b7f1ef1c18bf247fd9ccacb193"
                    />
                    <h3>
                              Alesso
                    </h3>
                    <h3>
                              WME
                    </h3>
                    <h3>
                              81
                    </h3>
          </div>
          <div
                    className="table-row"
          >
                    <img
                              alt="artist"
                              className="artist-image"
                              src="https://i.scdn.co/image/8c0fbe0fb1c86825adeb18bf0a019a22827c4422"
                    />
                    <h3>
                              Booka Shade
                    </h3>
                    <h3>
                              WME
                    </h3>
                    <h3>
                              51
                    </h3>
          </div>
          <div
                    className="table-row"
          >
                    <img
                              alt="artist"
                              className="artist-image"
                              src="https://i.scdn.co/image/1cbc13e4131461d8631d527c955ba83dddde099d"
                    />
                    <h3>
                              Chuckie
                    </h3>
                    <h3>
                              WME
                    </h3>
                    <h3>
                              51
                    </h3>
          </div>
</div>,
      ],
      "className": "watchlist",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Watchlist",
        },
        "ref": null,
        "rendered": "Watchlist",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Image
</h3>,
            <h3>
              Artist
</h3>,
            <h3>
              Agency
</h3>,
            <h3>
              Popularity
</h3>,
          ],
          "className": "recent-table-headings",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Image",
            },
            "ref": null,
            "rendered": "Image",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Artist",
            },
            "ref": null,
            "rendered": "Artist",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Agency",
            },
            "ref": null,
            "rendered": "Agency",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Popularity",
            },
            "ref": null,
            "rendered": "Popularity",
            "type": "h3",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="table-row"
>
              <img
                            alt="artist"
                            className="artist-image"
                            src="https://i.scdn.co/image/021f014a6809e5b7f1ef1c18bf247fd9ccacb193"
              />
              <h3>
                            Alesso
              </h3>
              <h3>
                            WME
              </h3>
              <h3>
                            81
              </h3>
</div>,
            <div
              className="table-row"
>
              <img
                            alt="artist"
                            className="artist-image"
                            src="https://i.scdn.co/image/8c0fbe0fb1c86825adeb18bf0a019a22827c4422"
              />
              <h3>
                            Booka Shade
              </h3>
              <h3>
                            WME
              </h3>
              <h3>
                            51
              </h3>
</div>,
            <div
              className="table-row"
>
              <img
                            alt="artist"
                            className="artist-image"
                            src="https://i.scdn.co/image/1cbc13e4131461d8631d527c955ba83dddde099d"
              />
              <h3>
                            Chuckie
              </h3>
              <h3>
                            WME
              </h3>
              <h3>
                            51
              </h3>
</div>,
          ],
          "className": "offers-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "4",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt="artist"
                  className="artist-image"
                  src="https://i.scdn.co/image/021f014a6809e5b7f1ef1c18bf247fd9ccacb193"
/>,
                <h3>
                  Alesso
</h3>,
                <h3>
                  WME
</h3>,
                <h3>
                  81
</h3>,
              ],
              "className": "table-row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "artist",
                  "className": "artist-image",
                  "src": "https://i.scdn.co/image/021f014a6809e5b7f1ef1c18bf247fd9ccacb193",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Alesso",
                },
                "ref": null,
                "rendered": "Alesso",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "WME",
                },
                "ref": null,
                "rendered": "WME",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": 81,
                },
                "ref": null,
                "rendered": 81,
                "type": "h3",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "18",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt="artist"
                  className="artist-image"
                  src="https://i.scdn.co/image/8c0fbe0fb1c86825adeb18bf0a019a22827c4422"
/>,
                <h3>
                  Booka Shade
</h3>,
                <h3>
                  WME
</h3>,
                <h3>
                  51
</h3>,
              ],
              "className": "table-row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "artist",
                  "className": "artist-image",
                  "src": "https://i.scdn.co/image/8c0fbe0fb1c86825adeb18bf0a019a22827c4422",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Booka Shade",
                },
                "ref": null,
                "rendered": "Booka Shade",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "WME",
                },
                "ref": null,
                "rendered": "WME",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": 51,
                },
                "ref": null,
                "rendered": 51,
                "type": "h3",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "23",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt="artist"
                  className="artist-image"
                  src="https://i.scdn.co/image/1cbc13e4131461d8631d527c955ba83dddde099d"
/>,
                <h3>
                  Chuckie
</h3>,
                <h3>
                  WME
</h3>,
                <h3>
                  51
</h3>,
              ],
              "className": "table-row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "artist",
                  "className": "artist-image",
                  "src": "https://i.scdn.co/image/1cbc13e4131461d8631d527c955ba83dddde099d",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Chuckie",
                },
                "ref": null,
                "rendered": "Chuckie",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "WME",
                },
                "ref": null,
                "rendered": "WME",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": 51,
                },
                "ref": null,
                "rendered": 51,
                "type": "h3",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Watchlist
</h2>,
          <div
            className="recent-table-headings"
>
            <h3>
                        Image
            </h3>
            <h3>
                        Artist
            </h3>
            <h3>
                        Agency
            </h3>
            <h3>
                        Popularity
            </h3>
</div>,
          <div
            className="offers-container"
>
            <div
                        className="table-row"
            >
                        <img
                                    alt="artist"
                                    className="artist-image"
                                    src="https://i.scdn.co/image/021f014a6809e5b7f1ef1c18bf247fd9ccacb193"
                        />
                        <h3>
                                    Alesso
                        </h3>
                        <h3>
                                    WME
                        </h3>
                        <h3>
                                    81
                        </h3>
            </div>
            <div
                        className="table-row"
            >
                        <img
                                    alt="artist"
                                    className="artist-image"
                                    src="https://i.scdn.co/image/8c0fbe0fb1c86825adeb18bf0a019a22827c4422"
                        />
                        <h3>
                                    Booka Shade
                        </h3>
                        <h3>
                                    WME
                        </h3>
                        <h3>
                                    51
                        </h3>
            </div>
            <div
                        className="table-row"
            >
                        <img
                                    alt="artist"
                                    className="artist-image"
                                    src="https://i.scdn.co/image/1cbc13e4131461d8631d527c955ba83dddde099d"
                        />
                        <h3>
                                    Chuckie
                        </h3>
                        <h3>
                                    WME
                        </h3>
                        <h3>
                                    51
                        </h3>
            </div>
</div>,
        ],
        "className": "watchlist",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Watchlist",
          },
          "ref": null,
          "rendered": "Watchlist",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Image
</h3>,
              <h3>
                Artist
</h3>,
              <h3>
                Agency
</h3>,
              <h3>
                Popularity
</h3>,
            ],
            "className": "recent-table-headings",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Image",
              },
              "ref": null,
              "rendered": "Image",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Artist",
              },
              "ref": null,
              "rendered": "Artist",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Agency",
              },
              "ref": null,
              "rendered": "Agency",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Popularity",
              },
              "ref": null,
              "rendered": "Popularity",
              "type": "h3",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="table-row"
>
                <img
                                alt="artist"
                                className="artist-image"
                                src="https://i.scdn.co/image/021f014a6809e5b7f1ef1c18bf247fd9ccacb193"
                />
                <h3>
                                Alesso
                </h3>
                <h3>
                                WME
                </h3>
                <h3>
                                81
                </h3>
</div>,
              <div
                className="table-row"
>
                <img
                                alt="artist"
                                className="artist-image"
                                src="https://i.scdn.co/image/8c0fbe0fb1c86825adeb18bf0a019a22827c4422"
                />
                <h3>
                                Booka Shade
                </h3>
                <h3>
                                WME
                </h3>
                <h3>
                                51
                </h3>
</div>,
              <div
                className="table-row"
>
                <img
                                alt="artist"
                                className="artist-image"
                                src="https://i.scdn.co/image/1cbc13e4131461d8631d527c955ba83dddde099d"
                />
                <h3>
                                Chuckie
                </h3>
                <h3>
                                WME
                </h3>
                <h3>
                                51
                </h3>
</div>,
            ],
            "className": "offers-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "4",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="artist"
                    className="artist-image"
                    src="https://i.scdn.co/image/021f014a6809e5b7f1ef1c18bf247fd9ccacb193"
/>,
                  <h3>
                    Alesso
</h3>,
                  <h3>
                    WME
</h3>,
                  <h3>
                    81
</h3>,
                ],
                "className": "table-row",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "artist",
                    "className": "artist-image",
                    "src": "https://i.scdn.co/image/021f014a6809e5b7f1ef1c18bf247fd9ccacb193",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Alesso",
                  },
                  "ref": null,
                  "rendered": "Alesso",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "WME",
                  },
                  "ref": null,
                  "rendered": "WME",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": 81,
                  },
                  "ref": null,
                  "rendered": 81,
                  "type": "h3",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "18",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="artist"
                    className="artist-image"
                    src="https://i.scdn.co/image/8c0fbe0fb1c86825adeb18bf0a019a22827c4422"
/>,
                  <h3>
                    Booka Shade
</h3>,
                  <h3>
                    WME
</h3>,
                  <h3>
                    51
</h3>,
                ],
                "className": "table-row",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "artist",
                    "className": "artist-image",
                    "src": "https://i.scdn.co/image/8c0fbe0fb1c86825adeb18bf0a019a22827c4422",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Booka Shade",
                  },
                  "ref": null,
                  "rendered": "Booka Shade",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "WME",
                  },
                  "ref": null,
                  "rendered": "WME",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": 51,
                  },
                  "ref": null,
                  "rendered": 51,
                  "type": "h3",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "23",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="artist"
                    className="artist-image"
                    src="https://i.scdn.co/image/1cbc13e4131461d8631d527c955ba83dddde099d"
/>,
                  <h3>
                    Chuckie
</h3>,
                  <h3>
                    WME
</h3>,
                  <h3>
                    51
</h3>,
                ],
                "className": "table-row",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "artist",
                    "className": "artist-image",
                    "src": "https://i.scdn.co/image/1cbc13e4131461d8631d527c955ba83dddde099d",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Chuckie",
                  },
                  "ref": null,
                  "rendered": "Chuckie",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "WME",
                  },
                  "ref": null,
                  "rendered": "WME",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": 51,
                  },
                  "ref": null,
                  "rendered": 51,
                  "type": "h3",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
